### DISCUSSION TITLE
google earth engine web application deployment

### DISCUSSION THREAD

**Discussion by Animesh-na** [opengeos/leafmap#102] [Q&A] [answered]
I have deployed a sample web application https://damp-stream-22347.herokuapp.com/ as describe in tutorial-26 geemap. can we use simply python for making a dynamic web application with geemap.  can we use parameters dynamically like coordinates and images and results (images thumbnails, maps) stored in a database.
Thank you!

**Comment by giswqs** [comment, marked as answer]
Yes, behind the scene of the web app, you can do whatever you want to process and store the data. See https://geemap.org/notebooks/39_timelapse/
